test/integration/consul-container/test/troubleshoot/troubleshoot_proxy_test.go

```diff
@@ -0,0 +1,73 @@
+package troubleshoot
+
+import (
+	"context"
+	"fmt"
+	libcluster "github.com/hashicorp/consul/test/integration/consul-container/libs/cluster"
+	libservice "github.com/hashicorp/consul/test/integration/consul-container/libs/service"
+	"github.com/hashicorp/consul/test/integration/consul-container/libs/topology"
+	"github.com/hashicorp/consul/test/integration/consul-container/test/observability"
+	"github.com/stretchr/testify/require"
+	"io"
+	"testing"
+	"time"
+)
+
+func TestTroubleshootProxy_Success(t *testing.T) {
```

Comment:
You can add `t.Parrallel` on all of these. It's just missing from the AccessLog test by accident.

Reply:
with parallel I had some tests failing. I think there's a race condition in createCluster. But I will address your previous comment and look into the performance.

Reply:
@DanStough , given more tests have been added to consul-container, shall we increase the number of parallel jobs (i think the current value is `p=4`)?

Reply:
@huikang Sure. RB wrote some code similar to `freeport` which will throttle the tests when docker networks aren't available, so it may not increase the speed of the tests at some point.